index.d.ts

```diff
@@ -62,3 +62,39 @@ export class Readability<T = string> {
     publishedTime: string | null | undefined;
   };
 }
+
+// Assuming Article is the return type of Readability.prototype.parse()
+export type Article = ReturnType<Readability['parse']>;
+
+export interface ReadabilityOptions {
```

Comment:
Shouldn't this then be used in the definition of the constructor, above? It also seems like the definition here does not match the definition there, e.g. `nbTopCandidates` is missing here and `headless` is added...